rasa/cli/scaffold.py

```diff
@@ -127,21 +131,22 @@ def print_run_or_instructions(args: argparse.Namespace) -> None:
 def init_project(args: argparse.Namespace, path: Text) -> None:
     """Inits project."""
     os.chdir(path)
-    create_initial_project(".")
+    create_initial_project(".", args.dm2)
     print(f"Created project directory at '{os.getcwd()}'.")
     print_train_or_instructions(args)
 
 
-def create_initial_project(path: Text) -> None:
+def create_initial_project(path: Text, is_dm2: bool=False) -> None:
     """Creates directory structure and templates for initial project."""
     from distutils.dir_util import copy_tree
 
-    copy_tree(scaffold_path(), path)
+    copy_tree(scaffold_path(is_dm2), path)
 
 
-def scaffold_path() -> Text:
+def scaffold_path(is_dm2: bool=False) -> Text:
     import pkg_resources
-
+    if is_dm2:
+        return pkg_resources.resource_filename(__name__, "initial_project_dm2")
```

Comment:
resource needs to be added to `pyproject.toml`

Reply:
thanks for calling that out. Where though? I don't see anytihng like that for `initial_project`

Reply:
should be part of this line, but it is very intersting that the original one isn't https://github.com/RasaHQ/rasa/blob/b5af522a48d09783e356cc99d9237e8d3c28be8e/pyproject.toml#L21

it looks like, by default already anything in the module tree is added in the sdist case https://github.com/python-poetry/poetry/issues/2015#issuecomment-585841584